drivers/virtualization/virt_ivshmem.c

```diff
@@ -224,8 +226,6 @@ static int ivshmem_init(const struct device *dev)
 {
 	struct ivshmem *data = dev->data;
 
-	data->bdf = pcie_bdf_lookup(PCIE_ID(IVSHMEM_VENDOR_ID,
```

Comment:
You'll need to keep that one. But under the condition that data->bdf == PCIE_BDF_NONE (so in the dts node reg will be: reg = <PCIE_BDF_NONE PCIE_ID(0x1af4,0x1110)>;

Reply:
You'll be able to test this directly with tests/drivers/virtualization/ivshmem, add a DTS overlay with the ivshmem node owning such reg. Should work